dds/DCPS/DataReaderImpl.cpp

```diff
@@ -50,32 +50,30 @@ OPENDDS_BEGIN_VERSIONED_NAMESPACE_DECL
 namespace OpenDDS {
 namespace DCPS {
 
+#ifndef OPENDDS_NO_OWNERSHIP_KIND_EXCLUSIVE
+
```

Comment:
This macro block is empty, was something supposed to go here?

Reply:
It should be deleted. Don't remember why it's there in the first place.